projects/core/src/user/facade/user.service.ts

```diff
@@ -96,114 +49,8 @@ export class UserService {
-    );
-  }
-
-  /**
-   * Returns the register user process success flag.
-   *
-   * @deprecated since 3.2, subscribe to `UserRegisterFacade.register()` from `@spartacus/user` package
-   * to get the success state.
-   */
-  getRegisterUserResultSuccess(): Observable<boolean> {
-    return this.store.pipe(
-      select(getProcessSuccessFactory(REGISTER_USER_PROCESS_ID))
-    );
-  }
-
-  /**
-   * Returns the register user process error flag
-   *
-   * @deprecated since 3.2, subscribe to `UserRegisterFacade.register()` from `@spartacus/user` package
-   * to get the error state.
-   */
-  getRegisterUserResultError(): Observable<boolean> {
-    return this.store.pipe(
-      select(getProcessErrorFactory(REGISTER_USER_PROCESS_ID))
-    );
-  }
-
-  /**
-   * Resets the register user process flags
-   *
-   * @deprecated since 3.2, no longer needed when you use `UserRegisterFacade.register()`
-   * from `@spartacus/user` package.
-   */
-  resetRegisterUserProcessState(): void {
-    return this.store.dispatch(new UserActions.ResetRegisterUserProcess());
-  }
-
-  /**
-   * Remove user account, that's also called close user's account.
-   *
-   * @deprecated since 3.2, use `UserProfileFacade.close()` from `@spartacus/user` package.
-   */
-  remove(): void {
-    this.userIdService.invokeWithUserId((userId) => {
-      this.store.dispatch(new UserActions.RemoveUser(userId));
-    });
-  }
-
-  /**
-   * Returns the remove user loading flag.
-   *
-   * @deprecated since 3.2, subscribe to `UserProfileFacade.close()` from `@spartacus/user` package
-   * to get the loading state.
-   */
-  getRemoveUserResultLoading(): Observable<boolean> {
-    return this.store.pipe(
-      select(getProcessLoadingFactory(REMOVE_USER_PROCESS_ID))
-    );
-  }
-
-  /**
-   * Returns the remove user failure outcome.
-   *
-   * @deprecated since 3.2, subscribe to `UserProfileFacade.close()` from `@spartacus/user` package
-   * to get the error state.
-   */
-  getRemoveUserResultError(): Observable<boolean> {
-    return this.store.pipe(
-      select(getProcessErrorFactory(REMOVE_USER_PROCESS_ID))
-    );
-  }
-
-  /**
-   * Returns the remove user process success outcome.
-   *
-   * @deprecated since 3.2, subscribe to `UserProfileFacade.close()` from `@spartacus/user` package
-   * to get the success state.
-   */
-  getRemoveUserResultSuccess(): Observable<boolean> {
-    return this.store.pipe(
-      select(getProcessSuccessFactory(REMOVE_USER_PROCESS_ID))
-    );
-  }
-
-  /**
-   * Resets the remove user process state. The state needs to be reset after the process
-   * concludes, regardless if it's a success or an error
-   *
-   * @deprecated since 3.2, no longer needed when you use `UserProfileFacade.close()`
-   * from `@spartacus/user` package.
-   *
-   */
-  resetRemoveUserProcessState(): void {
-    this.store.dispatch(new UserActions.RemoveUserReset());
+    throw Error('Cannot get a user. `UserRegisterFacade` was not provided.');
   }
 
   /**
```

Comment:
Why don't we remove the titles from ngrx as well?

Reply:
I removed titles from ngrx.